ir/visitor.h

```diff
@@ -312,7 +314,7 @@ class Transform : public virtual Visitor {
 class ControlFlowVisitor : public virtual Visitor {
     std::map<const IR::Node *, std::pair<ControlFlowVisitor *, int>> *flow_join_points = 0;
  protected:
-    virtual ControlFlowVisitor *clone() const = 0;
+    ControlFlowVisitor* clone() const override;
```

Comment:
Is it not possible to define this as a abstract method? (`= 0;`).  If not, its probably better to remove it completely rather than overriding Visitor::clone with an identical method.

Reply:
fixed.